packages/core/src/components/KeyboardAvoidingView.tsx

```diff
@@ -51,7 +51,7 @@ const KeyboardAvoidingView: React.FC<KeyboardAvoidingViewProps> = ({
 
   return (
     <KeyboardAvoidingViewComponent
-      behavior={behaviorResult}
+      behavior={behaviorResult ?? undefined}
```

Comment:
This is an optional value, so I think they can just not pass anything and it would be equivalent to passing null? Or am I missing something?